src/components/safe-apps/AddCustomAppModal.tsx

```diff
@@ -114,13 +111,20 @@ const AddCustomAppModal = ({ open, onClose, onSave, safeAppsList }: Props) => {
               mt: 2,
             }}
           >
-            <ImageFallback
-              height={TEXT_FIELD_HEIGHT}
-              src={appLogoUrl}
-              fallbackSrc={APP_LOGO_FALLBACK_IMAGE}
-              alt="Apps icon"
+            <SvgIcon component={SafeAppIcon} inheritViewBox sx={{ height: TEXT_FIELD_SIZE, width: TEXT_FIELD_SIZE }} />
+            <TextField
+              label="App name"
+              disabled
+              sx={({ palette }) => ({
+                width: '100%',
+                ml: 2,
+                WebkitTextFillColor: palette.text.secondary,
+                '&& input': {
+                  WebkitTextFillColor: safeApp?.name && palette.text.primary,
```

Comment:
What's this for? And why only webkit?

Reply:
hey @katspaugh !! 

This configuration is in order to make the app name title standout a little as the textfield is disabled. So it looks like an enabled one in both themes

![image](https://user-images.githubusercontent.com/1576776/193761191-8cd9642b-1b8f-4e1a-bb4b-b075ab711da1.png)

![image](https://user-images.githubusercontent.com/1576776/193761295-efbda8a4-76c8-4c51-bd41-647980afc026.png)


Webkit is because is a non standard property. Don't know if you know a better way to do it

Reply:
If it's disabled, it should look like one. Otherwise the user might try to edit it, resulting in a bad UX.
Maybe you can use `readOnly` instead of `disabled`?

Reply:
In this concrete case for me it looks better without leaving the default disable colors

![image](https://user-images.githubusercontent.com/1576776/193762753-0ba9ef32-7a9f-44c5-8471-8eaea08dff58.png)

The problem with readOnly is that the adornment is going up and down when clicking over the field and the colors change on blur. Disable for me fits better

Reply:
Hey @katspaugh we were talking in the daily about this and we agree about bringing @liliiaorlenko into the conversation for make some design change and substitute this input for a label/text/any. It doesn't make sense to have a disabled input there and is more an historical thing

Lila, could you provide us with some guidelines to change The App Name from an input to something fancier 😁 ?

Reply:
I made a designs with different state for this screen https://www.figma.com/file/rEKs2w2zMdmhNtkgTQDEWD/Add-custom-app?node-id=2%3A2757
- start screen
- app can be added
- error

Reply:
Let's try then! Thanks

Will ping you again when implemented

Reply:
Hey guys, we are going to create a new ticket for the redesign of the popup and send this PR to QA as it fix another things